src/main/java/seedu/address/ui/CalendarView.java

```diff
@@ -0,0 +1,270 @@
+package seedu.address.ui;
+
+import java.time.LocalDate;
+import java.time.YearMonth;
+
+import javafx.fxml.FXML;
+import javafx.geometry.HPos;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.geometry.VPos;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.effect.BlendMode;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.Background;
+import javafx.scene.layout.BackgroundFill;
+import javafx.scene.layout.Border;
+import javafx.scene.layout.BorderStroke;
+import javafx.scene.layout.BorderStrokeStyle;
+import javafx.scene.layout.BorderWidths;
+import javafx.scene.layout.CornerRadii;
+import javafx.scene.layout.GridPane;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import javafx.scene.paint.Color;
+
+/**
+ * Calendar View UI.
+ */
+public class CalendarView extends UiPart<Region> {
+
+    private static final String FXML = "CalendarView.fxml";
+    private static final String[] MONTHS = {"January", "February", "March", "April", "May", "June",
+        "July", "August", "September", "October", "November", "December"};
+    private static final int TOTALNUMOFDATEGRID = 42;
+    private static final int NUMOFDAYSPERWEEK = 7;
```

Comment:
Should add underscore between the words. I believe this is part of the naming conventions

Reply:
Good point! I've changed these 2 in the latest commit. Thank you!